src/ray/gcs/gcs_server/gcs_actor_manager.cc

```diff
@@ -95,6 +95,15 @@ const rpc::ActorTableData &GcsActor::GetActorTableData() const {
 
 rpc::ActorTableData *GcsActor::GetMutableActorTableData() { return &actor_table_data_; }
 
+std::shared_ptr<GcsActorWorkerAssignment> GcsActor::GetActorWorkerAssignment() const {
```

Comment:
do you mean `std::shared_ptr<const GcsActorWorkerAssignment>`?

Reply:
Yes, fixed.